### ISSUE TITLE
Script doesn't have access to files on Machine (GCP 2016 - 2019)

### ISSUE THREAD

**Issue by Maladril** [jamesstringerparsec/Parsec-Cloud-Preparation-Tool#24]
Script doesn't run, seems like it doesn't have permission to extract the files or access them (GCP, tried Server 2016 and 2019, new VMs, no difference, both fail)

I tried manually extracting the files from the zip and re-running, still fails.

![image](https://user-images.githubusercontent.com/21103893/78682654-441a9900-78bc-11ea-8985-d0263124867c.png)

**Comment by Maladril** [comment]
Extracting the files manually got me further but still having errors:

![image](https://user-images.githubusercontent.com/21103893/78683166-f6eaf700-78bc-11ea-92bd-f08a9cbf43a3.png)

**Comment by jamesstringerparsec** [comment]
You entered the Powershell lines in one by one, but in the wrong order.

**Comment by Maladril** [comment]
*facepalm* thanks. I can't understand how copy-paste can fail so bad... I didn't enter one by one, all as a block...
After investigating, it seems that Chrome RDP recently changed and mess-up the copy-pasting and puts it in reverse order.

I suggest always copy-pasting from the local computer browser for the script... lesson learned...

I checked to make sure I didn't have something like right to left typing enabled or something, I still can't find what changed since I did THE EXACT SAME THING 2 weeks ago and everything went fine.
